app/models/katello/glue/pulp/repo.rb

```diff
@@ -502,6 +501,7 @@ def find_packages_by_nvre(name, version, release, epoch)
       end
 
       def pulp_update_needed?
+        return true if ostree?
```

Comment:
The implication here is that redhat or not, we allow changing ostree repositories?